angio-frontend/src/modules/common/components/BaseSubheader.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <v-layout mb-4>
+        <h3 class="display-1 font-weight-regular">{{ subheader }}</h3>
+    </v-layout>
+</template>
+
+<script lang="ts">
+    import {Component, Prop, Vue} from "vue-property-decorator";
+
+    @Component
+    export default class BaseSubheader extends Vue {
+
+        @Prop()
+        public readonly subheader!: string;
```

Comment:
The component is already named "subheader", so it would be better to name the prop, for example, "value" or "text"

Reply:
Done